vaultaire-common.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       >= 1.10
 name:                vaultaire-common
-version:             2.8.4
+version:             2.8.5
```

Comment:
That's a patch bump - minor bump is to 2.9.0 (semver standard is 'major.minor.patch.[otherthings]').